docs/.htmltest.yml

```diff
@@ -4,6 +4,7 @@ IgnoreDirectoryMissingTrailingSlash: true
 IgnoreDirs:
   - favicons
   - docs/crd-ref/lifecycle/
+  - community/
```

Comment:
why did you fully ignore this folder?

Reply:
Because `community` will be filled from another repository. Based on that you have no control within your pull request over the content of this directory, and it should not block you.